packages/react-core/src/filters/Filter.js

```diff
@@ -1,7 +1,13 @@
 import { filterFunctions } from './FilterTypes';
+import { FiltersLogicalOperators } from '../operations/constants/FiltersLogicalOperators';
 
-function passesFilter(columns, filters, feature) {
-  return columns.every((column) => {
+function passesFilter(columns, filters, feature, filtersLogicalOperator) {
+  const logicalOperatorMethods = {
+    [FiltersLogicalOperators.AND]: 'every',
+    [FiltersLogicalOperators.OR]: 'some'
+  };
```

Comment:
It's a constant, take it out of the function to avoid instantiate it every time.

Reply:
Good catch !